brvs-core/src/main/java/iroha/validation/rest/RestService.java

```diff
@@ -98,6 +98,22 @@ public Response sendTransactionSign(String transaction) {
     return sendTransactionWithSigning(transaction);
   }
 
+  @POST
+  @Path("/transaction/sendBinary")
+  @Consumes(MediaType.APPLICATION_JSON)
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response sendTransactionNoSign(byte[] transaction) {
+    return sendBinaryTransaction(transaction, false);
```

Comment:
Could you please refactor it a little bit to get rid of boolean arg? Please take a look at `sendTransactionSign` and `sendTransactionNoSign`.